ion-schema/src/isl/version_based_isl/constraints/not.rs

```diff
@@ -0,0 +1,16 @@
+use crate::isl::version_based_isl::{IslTypeArgument, IslVersionTrait};
+
+#[derive(Debug, Clone, PartialEq)]
+pub struct Not<V: IslVersionTrait> {
```

Comment:
Add `impl <V: IslVersionTrait> From<Not<V>> for IslConstraint<V>`, and similarly for all other constraints.